examples/basic/main.tf

```diff
@@ -0,0 +1,17 @@
+########################################################################################################################
+# MAS Core on an existing Openshift cluster
```

Comment:
examples are supposed to be end to end and include all dependencies. I suggest you add a very simple cluster to this basic example to make it end to end. But it can be done as another PR so we don;t hold this one up

Reply:
Agree..will do this after the GA..thanks